package.json

```diff
@@ -21,6 +21,10 @@
     "build": "tsc --noEmit --project .",
     "buildDocs": "typedoc"
   },
+  "dependencies": {
+    "@ethereumjs/tx": "^3.2.1",
+    "eth-sig-util": "^3.0.1"
```

Comment:
I would like to replace the two types from these packages with our own. Better to avoid any non-type dependencies if possible. Plus, we ought to have our own Transaction type.

Reply:
To confirm, the types to develop internally are,

1. `@ethereumjs/tx` - `Transaction` ([source](https://github.com/ethereumjs/ethereumjs-tx/blob/master/docs/classes/transaction.md))
2. `eth-sig-util` - `EthEncryptedData` ([source](https://www.jsdocs.io/package/eth-sig-util#EthEncryptedData))